src/components/tasks/TasksContent.tsx

```diff
@@ -15,14 +15,16 @@ import { getActiveTab, tabToUrlParams } from '@/utils/getActiveTab';
 import { Select } from '../Select';
 import { getChangedStatusName } from '@/utils/getChangedStatusName';
 import useIntersection from '@/hooks/useIntersection';
+import TaskSearch from './TaskSearch/TaskSearch';
 
-type routerQueryParams = {
-    section?: string;
-};
-export const TasksContent = () => {
+const getQueryParamValue = (tab: Tab) => `is:${tabToUrlParams(tab)}`;
+
+export const TasksContent = ({ dev }: { dev: boolean }) => {
     const router = useRouter();
-    const { section }: routerQueryParams = router.query;
-    const selectedTab = getActiveTab(section);
+    const allQueryParams = router.query;
+    const q = allQueryParams.q as string;
+    const queryTab = q?.replace('is:', '');
```

Comment:
NIT - is it possible to rename `q` with some better naming?

Reply:
maybe `q-value` or `q-param-value` or `queryParam`

Reply:
Done